server/service/src/sync/integrate_document.rs

```diff
@@ -0,0 +1,87 @@
+use repository::{
```

Comment:
I am ignoring this in review, as I am not sure how it relates to document sync and doesn't seem to have any description of this in the PR.